articles/openshift/openshift-faq.md

```diff
@@ -10,196 +10,220 @@ ms.date: 11/04/2019
+No in Azure Red Hat OpenShift 3.11.
 
-Notifications may be sent via ICM and email if communication to the customer is required.
+### Can logs of underlying VMs be streamed out to a customer log analysis system?
 
-## What about emergency vs. planned maintenance windows?
+Logs from underlying VMs are handled by the managed service and are not exposed to customers.
 
-We do not distinguish between the two types of maintenance. Our teams are available 24/7/365 and do not use traditional scheduled “out-of-hours” maintenance windows.
+### How can a customer get access to metrics like CPU/memory at the node level to take action to scale, debug issues, etc. I cannot seem to run kubectl top on an Azure Red Hat OpenShift cluster.
 
-## How will host operating system and OpenShift software be updated?
+For Azure Red Hat OpenShift 4.x clusters, the web console contains all metrics at the node level. See the Red Hat documentation on [viewing cluster information](https://docs.openshift.com/aro/4/web_console/using-dashboard-to-get-cluster-information.html) for more details.
 
-The host operating system and OpenShift software are updated through our general upgrade and image build process.
+For Azure Red Hat OpenShift 3.11 clusters, customers can access the CPU/Memory metrics at the node level by using the command `oc adm top nodes` or `kubectl top nodes` with the customer-admin cluster role. Customers can also access the CPU/Memory metrics of `pods` with the command `oc adm top pods` or `kubectl top pods`.
 
-## What’s the process to reboot the updated node?
+### If we scale up the deployment, how do Azure fault domains map into pod placement to ensure all pods for a service do not get knocked out by a failure in a single fault domain?
 
-This should be handled as a part of an upgrade.
+There are by default five fault domains when using virtual machine scale sets in Azure. Each virtual machine instance in a scale set will get placed into one of these fault domains. This ensures that applications deployed to the compute nodes in a cluster will get placed in separate fault domains.
 
-## Is data stored in etcd encrypted on ARO?
+Refer to [Choosing the right number of fault domains for virtual machine scale set](https://docs.microsoft.com//azure/virtual-machine-scale-sets/virtual-machine-scale-sets-manage-fault-domains) for more details.
 
-It is not encrypted on the etcd level. The option to turn it on is currently unsupported. OpenShift supports this feature, but engineering efforts are required to make it on the road map. The data is encrypted at the disk level. Refer to [Encrypting Data at Datastore Layer](https://docs.openshift.com/container-platform/3.11/admin_guide/encrypting_data.html) for more information.
+### Is there a way to manage pod placement?
```

Comment:
Should we talk about taints and tolerations here?

Reply:
I'm not sure what to say about this one. Per the discussion here (https://docs.google.com/document/d/1ECPGVyVrKfdUz0aVBlvWuhFnLynT6GRqApGoQGpgw2I/edit#heading=h.kg3tgr8oa951) you should be able to do anything you can do in OCP.

I'll add the following link for now: https://docs.openshift.com/aro/4/nodes/scheduling/nodes-scheduler-about.html